internal/core/keeper/registry/registry_test.go

```diff
@@ -0,0 +1,422 @@
+//
+// Copyright (C) 2024 IOTech Ltd

```

Comment:
Please update the copyright header to the correct year
```suggestion
// Copyright (C) 2025 IOTech Ltd

```